cli/src/mctl.rs

```diff
@@ -172,12 +176,13 @@ async fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
         Sub::Publish {
             uuid,
             key,
+            protocol,
         } => serde_json::to_string_pretty(
             &call::<_, PublishNexusReply>(
                 &opt.socket,
                 "publish_nexus",
                 Some(json!({ "uuid": uuid,
-                    "share" : ShareProtocol::Nbd as i32,
+                    "share" : protocol,
```

Comment:
This will not work and indeed it does not when I port this change to my branch, the wire repr for "share" is i32 so change to protocol as i32.

Reply:
This does not work on the cli branch either.